test/unit/utils_test.js

```diff
@@ -1,26 +1,26 @@
 const randomBytes = require("randombytes");
```

Comment:
Prettier made a lot of formatting changes in this file, is that OK?

Reply:
Its just a bit harder to see what actually changed

Reply:
Could you break the formatting changes out to a separate PR so there's a clear distinction between those non-functional changes and the changes needed for SEP-10 v3.0.0?

Reply:
❗ Could you break the formatting changes out to a separate PR so there's a clear distinction between those non-functional changes and the changes needed for SEP-10 v3.0.0?

Reply:
Done, I disabled formatting for that file completely to keep it consistent with other files. Apologies for the initial mess!